modules/releng.gradle

```diff
@@ -343,6 +344,80 @@ prepareMajorIncrement {
 	}
 }
 
+tagAppReleases {
+	description = "Gathers the Git IDs from all app changelog files and tags them as 'marketplace-appName-appVersion'."
```

Comment:
Call it "change logs" instead of "changelog files", I think Brian wants "change log" to be 2 words.